src/main/java/core/basesyntax/interfaces/DrawAble.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface DrawAble {
+    void figureDraw();
+}
```

Comment:
```suggestion
public interface Drawable {
    void draw();
}
```

same, it's better not to use word 'figure' in interface name or method names as we can use this interface not only for figures